assets/js/base/context/cart-checkout/payment-methods/payment-method-data-context.js

```diff
@@ -103,26 +104,24 @@ const isErrorResponse = ( response ) => {
  * @param {Object} props                     Incoming props for provider
  * @param {Object} props.children            The wrapped components in this
  *                                           provider.
- * @param {string} props.activePaymentMethod The initial active payment method
- *                                           to set for the context.
  */
-export const PaymentMethodDataProvider = ( {
-	children,
-	activePaymentMethod: initialActivePaymentMethod,
-} ) => {
+export const PaymentMethodDataProvider = ( { children } ) => {
 	const { setBillingData } = useBillingDataContext();
 	const {
 		isComplete: checkoutIsComplete,
 		isProcessingComplete: checkoutIsProcessingComplete,
 		isCalculating: checkoutIsCalculating,
 		hasError: checkoutHasError,
 	} = useCheckoutContext();
-	const [ activePaymentMethod, setActive ] = useState(
-		initialActivePaymentMethod
-	);
+	const [ activePaymentMethod, setActive ] = useState( {} );
```

Comment:
You changed the default for the state. It should be an empty string. This breaks the initial active payment method calculation.